src/qt/bitcoin.cpp

```diff
@@ -436,12 +436,13 @@ int GuiMain(int argc, char* argv[])
     Q_INIT_RESOURCE(bitcoin);
     Q_INIT_RESOURCE(bitcoin_locale);
 
+#if QT_VERSION >= 0x050600
+    QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+#endif
+
     BitcoinApplication app(*node, argc, argv);
```

Comment:
I'd rather move this after all `setAttribute`. This shouldn't cause any issues, right?